src/components/SideMenu/styled.elements.ts

```diff
@@ -16,7 +16,7 @@ export const OperationBadge = styled.span.attrs((props: { type: string }) => ({
   background-repeat: no-repeat;
   background-position: 6px 4px;
   font-size: 7px;
-  font-family: Verdana; // web-safe
+  font-family: Verdana, sans-serif; // web-safe
```

Comment:
Yes, this is important!

Reply:
This is what it looks like for me without it:

![image](https://user-images.githubusercontent.com/1161871/90772745-9633e800-e2ba-11ea-8151-4d9a7bfa6d44.png)